android-components/components/support/webextensions/src/main/java/mozilla/components/support/webextensions/ExtensionsProcessSpawningThresholdObserver.kt

```diff
@@ -5,15 +5,14 @@
 package mozilla.components.support.webextensions
 
 import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.cancel
 import kotlinx.coroutines.flow.distinctUntilChangedBy
 import mozilla.components.browser.state.store.BrowserStore
 import mozilla.components.lib.state.ext.flowScoped
 import mozilla.components.support.base.feature.LifecycleAwareFeature
 
 /**
- * Feature implementation that shows a prompt when the extensions process spawning has been
- * disabled due to the restart threshold being met.
+ * Observes the [BrowserStore] state for when the extensions process spawning threshold has been
+ * exceeded and a user should be prompted. This requires running in both the foreground and background.
```

Comment:
The process spawning has been disabled when we get there so the previous sentence wasn't incorrect.

```suggestion
 * Observes the [BrowserStore] state for when the extensions process spawning has been disabled
 * and the user should be prompted. This requires running in both the foreground and background.
```

Reply:
If the maximum is set to 5 then this occurs on the 6th killing of the process. This should be correct as the threshold has been exceeded. For this PR I'll revert it but in the followup refactor we should consider this.